packages/autorest.python/autorest/codegen/models/constant_type.py

```diff
@@ -75,6 +75,8 @@ def docstring_type(self, **kwargs: Any) -> str:
         return self.value_type.docstring_type(**kwargs)
 
     def type_annotation(self, **kwargs: Any) -> str:
+        if self.code_model.options["models_mode"] == "dpg":
```

Comment:
should we do literal for the past models as well?

Reply:
bump on this

Reply:
Let me have a try on the msrest models as well